docker/Dockerfile.ci

```diff
@@ -1,7 +1,7 @@
-FROM node:20-alpine
+FROM oven/bun:latest
```

Comment:
Always use a fixed version, ideally staying away from using `latest` whenever possible. The problem of not using a fixed version, is that we may find ourselves several months down the road with a broken build, just because bun introduced a breaking change out of nowhere.

Reply:
Great point, thanks!

Will fix.